internal/locate/region_request.go

```diff
@@ -393,8 +393,8 @@ type tryFollower struct {
 	stateBase
 	leaderIdx AccessIndex
 	lastIdx   AccessIndex
-	// fromOnNotLeader indicates whether the state is changed from onNotLeader.
-	fromOnNotLeader bool
+	// fromKnownLeader indicates whether the state is changed from onNotLeader.
+	fromKnownLeader bool
```

Comment:
If it's to represent a fallback from `accessKnownLeader` to `tryFollower`, naming it `fromAccessknownLeader` is more accurate and understandable.